packages/apps/helpdesk/src/screens/TicketFormScreen.js

```diff
@@ -16,267 +16,110 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-import React, {useState, useCallback} from 'react';
-import {StyleSheet, View} from 'react-native';
-import {
-  Button,
-  FormInput,
-  KeyboardAvoidingScrollView,
-  Picker,
-  Screen,
-  useThemeColor,
-  FormHtmlInput,
-  checkNullString,
-} from '@axelor/aos-mobile-ui';
+import React, {useCallback, useMemo, useEffect} from 'react';
 import {
   useSelector,
-  useDispatch,
   useTranslator,
-  DateInput,
+  useDispatch,
+  FormView,
 } from '@axelor/aos-mobile-core';
 import {createTicket, updateTicket} from '../features/ticketSlice';
-import {
-  ContactPartnerSearchBar,
-  CustomerSearchBar,
-  DurationFormInput,
-  ProgressFormInput,
-  ProjectSearchBar,
-  TicketTypeSearchBar,
-  UserSearchBar,
-} from '../components';
-import {Ticket} from '../types/';
-
-const PROJECT_KEY = 'project';
-const CUSTOMER_KEY = 'customerPartner';
-const CONTACT_KEY = 'contactPartner';
-
-const DEFAULT_TICKET_VALUE = {duration: 0};
-
-const isObjectMissingRequiredField = object => checkNullString(object?.subject);
+import {getCustomerbyId} from '../features/customerSlice';
+import {Ticket} from '../types';
 
 const TicketFormScreen = ({navigation, route}) => {
   const idTicket = route?.params?.idTicket;
-  const dispatch = useDispatch();
+
+  const _dispatch = useDispatch();
   const I18n = useTranslator();
-  const Colors = useThemeColor();
 
   const {ticket} = useSelector(state => state.ticket);
 
-  const [_ticket, setTicket] = useState(
-    idTicket != null ? ticket : DEFAULT_TICKET_VALUE,
-  );
-  const [disabledButton, setDisabledButton] = useState(
-    isObjectMissingRequiredField(_ticket),
-  );
-
-  const handleTicketFieldChange = useCallback((newValue, fieldName) => {
-    setTicket(current => {
-      if (current?.[fieldName] === newValue) {
-        return current;
-      }
-
-      const updatedObject = current == null ? {} : {...current};
-
-      updatedObject[fieldName] = newValue;
-
-      if (fieldName === PROJECT_KEY) {
-        updatedObject[CUSTOMER_KEY] = newValue?.clientPartner;
-        updatedObject[CONTACT_KEY] = newValue?.contactPartner;
-      }
-
-      setDisabledButton(isObjectMissingRequiredField(updatedObject));
-
-      return updatedObject;
-    });
-  }, []);
-
-  const updateTicketAPI = useCallback(() => {
+  const defaultValue = useMemo(() => {
+    const _default = {
+      duration: 0,
+    };
+
+    if (idTicket != null) {
+      return {
+        ...ticket,
+        duration: ticket.duration || _default.duration,
+        prioritySelect: {
+          title: Ticket.getPriority(ticket?.prioritySelect, I18n),
+          key: ticket.prioritySelect,
+        },
```

Comment:
Simplify this, disabled isValueItem on Priority select and only item key will be needed. 
Do not forget to adapt TicketPriorityPicker